src/home/pam_systemd_home.c

```diff
@@ -283,12 +284,12 @@ static int handle_generic_user_record_error(
 
         if (sd_bus_error_has_name(error, BUS_ERROR_HOME_ABSENT)) {
                 (void) pam_prompt(handle, PAM_ERROR_MSG, NULL,
-                                  "Home of user %s is currently absent, please plug in the necessary storage device or backing file system.", user_name);
+                                  N_("Home of user %s is currently absent, please plug in the necessary storage device or backing file system."), user_name);
```

Comment:
AFAIK, `N_()` macro means that the message is marked for translation but the translated string is not used in the place where `N_()` is called.

The macro that uses the translated string is `_()`.

Reply:
fixed